abnf/odata-aggregation-abnf.txt

```diff
@@ -194,8 +194,9 @@ traverseTrafo   = %s"traverse" OPEN
 
 recHierReference = BWS rootExpr BWS COMMA
                    BWS recHierQualifier BWS COMMA
-                   BWS propertyPath BWS
+                   BWS recHierPropertyPath BWS
 recHierQualifier = odataIdentifier
+recHierPropertyPath = *( ( complexProperty / entityNavigationProperty ) "/" ) primitiveProperty

```

Comment:
Is this sufficient? Or do we also allow annotations and type casts here?

Reply:
I think adding annotations and type casts could be problematic, because we specified that "This path MUST be well-defined for all instances in the input set, otherwise the request is rejected." Instance annotations and specialized type may not be given for all instances and would consequently yield errors. So, IMO the rule is just right as it is written now.